src/Microsoft.Dnx.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -127,19 +125,13 @@ public LibraryDescription GetDescription(LibraryRange libraryRange, FrameworkNam
                     dependencies = GetDependencies(package, targetFramework, targetLibrary: null);
                 }
 
-                return new LibraryDescription
-                {
-                    LibraryRange = libraryRange,
-                    Identity = new LibraryIdentity
-                    {
-                        Name = package.Id,
-                        Version = package.Version
-                    },
-                    Type = "Package",
-                    Dependencies = dependencies,
-                    Resolved = resolved,
-                    Compatible = compatible
-                };
+                return new Runtime.PackageDescription(
```

Comment:
Done. Had to rename a private nested class `PackageDescription` to `PackageMetadata`